src/view/template.js

```diff
@@ -89,7 +89,7 @@ Candy.View.Template = (function(self){
 
 	self.Message = {
 		pane: '<div class="message-pane-wrapper"><ul class="message-pane"></ul></div>',
-		item: '<li><small data-timestamp="{{timestamp}}">{{time}}</small><div>' +
+		item: '<li data-message-id="{{id}}"><small data-timestamp="{{timestamp}}">{{time}}</small><div>' +
```

Comment:
Just to avoid potential colision with any reserved name. Happy to change it if you're confident that `data-id` is ok. I'm not a front-end guy ;)